Dn02/src/Dn02.jl

```diff
@@ -3,17 +3,21 @@ module Dn02
 using SpecialFunctions
 using Distributions  # for Normal() and cdf()
 
+export erf_taylor, cdf_taylor, cdf_legendre, cdf_asymptotic, my_cdf
+
 include("taylor.jl")
 include("gauss_legendre.jl")
 include("asymptotic.jl")
 
 function my_cdf(x::Float64; 
-                  taylor_terms::Int=25, 
-                  asymptotic_terms::Int=5)
+                  taylor_terms::Int=10,
+                  gauss_terms::Int=12, 
+                  asymptotic_terms::Int=10)
 
     if x < 0
         return 1.0 - my_cdf(-x; 
```

Comment:
This formula will lead to large relative error for $x<-5$. You can use asymptotic expansion for those values.